code/modules/chemistry/Reagents-Base.dm

```diff
@@ -482,6 +497,15 @@
 		growth_tick.growth_rate += 0.4
 		growth_tick.health_change += 0.4
 
+	on_mob_life(var/mob/M, var/mult = 1)
+		if(!M) M = holder.my_atom
+		if(prob(55) && isliving(M))
+			var/mob/living/L = M
+			if (L.organHolder?.intestines?.synthetic)
+				M.HealDamage("All", 0.5 * mult, 0.5 * mult)
```

Comment:
Potassium is silly-easy to get, just one-click or eat a banana. Will need to look at balance here *a lot*.